Dockerfile

```
@@ -0,0 +1,27 @@
+FROM ubuntu:latest
+
+# Update package lists and install necessary dependencies
+RUN apt-get update && apt-get install -y \
+    python3 \
+    python3-pip \
+    python3.12-venv \
```

```suggestion
    python3-venv \
```

So upgrades are possible